.github/workflows/test.yml

```diff
@@ -0,0 +1,59 @@
+name: nodl_to_policy
+
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+
+jobs:
+  build-and-test:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os:
+          - ubuntu-20.04
+        ros_distribution:
+          - galactic
+    steps:
+      # actions/checkout is not used because of a known issue where `colcon` cannot be found
+      # https://github.com/ros-tooling/setup-ros/issues/75
+      - name: Setup ROS2
+        uses: ros-tooling/setup-ros@v0.2
+        with:
+          use-ros2-testing: true
+          required-ros-distributions: ${{ matrix.ros_distribution }}
+
+      - name: Test nodl_to_policy
+        uses: ros-tooling/action-ros-ci@v0.2
+        id: action_ros_ci
+        with:
+          package-name: nodl_to_policy
+          colcon-defaults: |
+            {
+              "test": {
+                "mixin": ["coverage-pytest"]
+              }
+            }
+          colcon-mixin-repository: https://raw.githubusercontent.com/colcon/colcon-mixin-repository/1ddb69bedfd1f04c2f000e95452f7c24a4d6176b/index.yaml
+          target-ros2-distro: ${{ matrix.ros_distribution }}
+
+      # `outputs.root_dir` is set to the relative path of the checked out git repository
+      - name: Find project root
+        id: find_project
+        run: echo "::set-output name=root_dir::$(vcs diff -s --repos . | cut -d ' ' -f 1)"
+
+      - name: Codecov report
+        uses: codecov/codecov-action@v1
```

Comment:
I'm of two minds about the Code coverage report.  While it is nice to have, it also increases our attack surface (as the [recent Codecov vulnerability](https://about.codecov.io/security-update/) demonstrates).

My inclination is to leave Codecov out of this repository, and just run the tests in the GitHub action.  @wjwwood, any thoughts?

Reply:
Yikes. I wasn't aware of that vulnerability. I agree that it's entirely optional, and can leave it out of this repository.

Reply:
I don't mind either way. Codecov is reacting to it, and there's no guarantee that any other platform we use might not have similar issues at some point. That being said, Codecov's value-added is optional.

Reply:
I've removed the Codecov action in bded47e. I've still kept the `coverage-pytest` mixin and the `.coveragerc` file though, because I think there's enough value for it to stick around in the test XMLs.